packages/dashboard/src/CdcDashboard.tsx

```diff
@@ -37,11 +37,100 @@ import './scss/main.scss'
 import '@cdc/core/styles/v2/main.scss'
 import AdvancedEditor from '@cdc/core/components/AdvancedEditor'
 
+export type APIFilter = {
+  apiEndpoint: string
+  valueSelector: string
+  textSelector: string
+}
+
+export type SharedFilter = {
```

Comment:
@adamdoe I think moving types out which are shared like this one makes sense. This one should go maybe into a /core/types folder as this could be used by any of the packages.